### ISSUE TITLE
[Feature request] Automatic Updates (macOS)

### ISSUE THREAD

**Issue by jmp-0x7C0** [mullvad/mullvadvpn-app#6007]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

First I'd like to thank the entire Mullvad team for creating the best VPN services available, with that I have one suggestion that would make using the Mullvad macOS app more frictionless:
The process of manually clicking update in the Mullvad app, then downloading and manually installing every new update is somewhat clunky and annoying. It is also less secure as many users (including myself) don't always immediately notice that an update is available, or don't immediately click download and install when the notice shows up. 

Most macOS users expect third party native macOS apps not distributed through the Mac App Store to provide the ability to automatically download and install updates. Most macOS applications use [Sparkle](https://sparkle-project.org) or similar frameworks to handle automatic updates for them which provides for a great user experience. Other apps resort to installing custom daemons just for this purpose which is less ideal. 

### Alternative solutions

Manually install updates.

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [X] Other

### Operating System

- [ ] Android
- [ ] iOS
- [ ] Windows
- [X] macOS
- [ ] Linux

**Comment by blanchardjeremy** [comment]
Yeah just want to weigh in here - feels really essential for security that updates are automatic. A lot of users aren't keeping track of this kind of thing or are going to ignore updates.

**Comment by LoganDark** [comment]
Hi, I just got a notification that my "privacy might be at risk with this unsupported app version" and maybe I wouldn't be using such an Unsupported App Version if updating didn't require me to manually go to the website, download a disk image, open the disk image, and run and click through the installer. I'm actually pretty annoyed that it's sending me notifications like this, so I turned all notifications off. I don't imagine that was the intention but I'm not gonna put up with nags like this.

<p align="center"><img width="370" src="https://github.com/user-attachments/assets/3edf030e-ffbb-42a0-be9c-31c9638ad545" /></p>

**Comment by MrChocolatine** [comment]
They are currently implementing in-app upgrades:

https://github.com/mullvad/mullvadvpn-app/blob/64862490c2781e4be3b8e77d5c297198fcb63d54/CHANGELOG.md?plain=1#L28-L29

For now the feature is marked as "Unreleased", but I bet they are going to release it soon.

**Comment by MarkusPettersson98** [comment]
We released in-app updates for macOS & Windows in the [2025.8 release](https://github.com/mullvad/mullvadvpn-app/releases/tag/2025.8)!
